# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

cytonActionGoal action_goal
cytonActionResult action_result
cytonActionFeedback action_feedback
